src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/StoreController.java

```diff
@@ -28,8 +43,20 @@ public class StoreController {
      * CREATE ROLE
      * -------------------
      */
-
-     //insert here your code
+        @Operation(summary = "creates a new store", description = "create a new store by entering the required data")
```

Comment:
eliminates this, to avoid conflicts with the Miguel developer's tasks.  (swagger)